synchro/synchrotest.py

```diff
@@ -184,6 +184,8 @@
     "TestUnifiedFindShutdownError.test_Concurrent_shutdown_error_on_find",
     "TestUnifiedInsertShutdownError.test_Concurrent_shutdown_error_on_insert",
     "TestUnifiedPoolClearedError.test_PoolClearedError_does_not_mark_server_unknown",
+    # This test requires synchronous method calls.
+    "TestAutomaticDecryptionKeys.test_01_simple_create",
```

Comment:
Why skip this test?

Reply:
We cannot guarantee that the `create_encrypted_collection` will happen before the `insert_one` unless we do some funky stuff with the PyMongo test--easier to just skip.

Reply:
What do you mean? First synchro runs create_encrypted_collection() then it runs insert_one(). That test should work just fine.

Reply:
This operation is sync in PyMongo, but not in Motor: https://github.com/mongodb/mongo-python-driver/blob/5b96757b0eb14fec16214f942112a6f4293f9fbb/test/test_encryption.py#L2734, so the write error was not getting raised.

Reply:
The operation is asynchronous when using Motor yes but this is the synchro module which is synchronous. The synchro tests run with the same synchronous api as PyMongo, it just happens to use motor under the hood to execute those api calls.

Reply:
Ahhh, right, synchro hurts my brain.

Reply:
I was also confused about that, but I think it is more clear now. I will change the synchro bindings to unwrap the error.